builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -398,6 +396,31 @@ node() {
   }
 }
 
+def attachTagstoLogGroup(config, environment_logical_id, credsId){
+  try {
+    def deploymentDescriptor = null
+    if( env_deployment_descriptor != null ){
+      deploymentDescriptor = env_deployment_descriptor
+    } else {
+      deploymentDescriptor = config['deployment_descriptor']
+    }
+    // If no descriptor present then simply making an empty one. The readYaml default behavior is to return empty string back that is harmful as Map not String is expected below
+    def doc = deploymentDescriptor ? readYaml(text: deploymentDescriptor) : [:]
+    def functionsElem = doc['functions']
+
+    // changing variable names appropriately
+    def serviceDomain = "${configLoader.INSTANCE_PREFIX}-${config['domain']}-${config['service']}"
```

Comment:
While we do this, can we get rid of weird naming convention that we are following (use of `-FN_` etc in function names) - we might need to update sbr.groovy and other related files that defines/enforces this convention.

Reply:
Done